src/containers/StructurePage/VersionSelector.tsx

```diff
@@ -56,7 +48,11 @@ const StyledSelectLabel = styled(SelectLabel, {
   },
 });
 
-const StickyVersionSelector = () => {
+const Wrapper = styled("div", {
+  base: { display: "flex", justifyContent: "center", marginBlockStart: "3xlarge" },
+});
```

Comment:
Hvorfor trenger vi to wrappere?

Reply:
Tenker du på `StyledVersionSelector` og `Wrapper`?

Reply:
Ja. Kan vi ikke slå de sammen til ett element?

Reply:
`StyledVersionSelector` er også et litt misvisende navn. Det er jo ikke en stylet versjon av `VersionSelector`, tross alt.

Reply:
Nei, `StyledVersionSelector` legger jo på bakgrunn blant annet, `Wrapper` er for å posisjonere elementet

Reply:
Så ikke den andre kommentaren din når jeg svarte her, kan godt oppdatere navngivningen